DashAI/back/job/model_job.py

```diff
@@ -114,6 +116,16 @@ def run(self) -> None:
                 ) from e
 
             try:
+                splits = json.loads(experiment.splits)
+                if splits["has_changed"]:
+                    new_splits = {
+                        "train": splits["train"],
+                        "test": splits["test"],
+                        "validation": splits["validation"],
+                    }
+                    loaded_dataset = update_dataset_splits(
+                        loaded_dataset, new_splits, splits["is_random"]
+                    )
```

Comment:
When might the splits be updated?
I think that for a given experiment, it should always be the same indices.

Reply:
This is implemented this way because the datasets come already with a separation by default, so if the train, test and validation splits aren't changed by the user the code doesn't update the splits, but if when creating a new experiment the user wants a new combination for them different to the one setted in the dataset upload, the the code uploads the dataset splits.

This could eventually be deleted if the dataset isn't separated anymore when loading it and is only divided when configurating the experiment, so the difference between both cases will no longer exist.

Reply:
Ok!